spinalcordtoolbox/scripts/sct_register_to_template.py

```diff
@@ -630,6 +665,7 @@ def main(argv: Sequence[str]):
             '-o', add_suffix(ftmp_data, '_straightAffine'),
             '-d', ftmp_template,
             '-w', 'warp_curve2straightAffine.nii.gz',
+            '-x', 'linear',
```

Comment:
I notice the batch_processing.sh results have changed. Skimming the PR, I think 99% of your changes would have no effect on the other registration types, but I noticed the addition of `-x linear` here. 

The default value is `spline`... is there a reason for this change? I couldn't find any reasoning in the source commit from #4681: https://github.com/spinalcordtoolbox/spinalcordtoolbox/commit/16428885c1d4005ec9f66cf82b5f17070ee7ad20

Reply:
done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4647/commits/46c682c7a86cc65154afa8ecf06979cfb4cee226